tes/models.py

```diff
@@ -60,15 +56,15 @@ def strconv(value: Any) -> Any:
 # since an int64 value is encoded as a string in json we need to handle
 # conversion
 def int64conv(value: Optional[str]) -> Optional[int]:
-    if value is not None:
-        return int(value)
-    return value
+    if value is None:
+        return value
+    return int(value)
```

Comment:
No functional changes here. This one (and the next right below for `timestampconv()` I just reordered for improved readability (at least in my opinion)